zazu/util.py

```diff
@@ -95,20 +95,30 @@ def prompt(text, default=None, expected_type=str):
     return expected_type(result)
 
 
-def pick(choices, message):
+def pick(choices, message, checkbox=False):
```

Comment:
I like the idea of calling the checkbox flag 'multiple'...as in allow multiple things to be selected.